mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -365,7 +373,9 @@
     <!-- Preference for changing default theme to dark or light mode -->
     <string name="preferences_theme">Theme</string>
     <!-- Preference for customizing the home screen -->
-    <string name="preferences_home">Home</string>
+    <string moz:removedIn="94" name="preferences_home" tools:ignore="UnusedResources">Home</string>
```

Comment:
@eliserichards Hi! Could you remind me what the added attributes like the one above does on your side?
For us, these strings still appear in Pontoon (our localization tool) and the added attributes don't have any effect. So, wanted to make extra sure we are on the same page on what happens!

Reply:
If it has the `moz:RemovedIn` tag then it means that we are not going to be using it moving forward. In most cases, it would mean that we changed/updated the copy to something new. We add this tag so that we can deprecate it safely and not break anything in the current version. 

So if both `moz:RemovedIn` and `unusedResources` are there, it does NOT need to be translated because we are going to remove it.

Reply:
Thanks for all the details Elise! Very helpful. Yes as mentioned on Slack, as long as these stay in the strings file, they stay exposed for localization in our side on Pontoon
@flodolo Since we talked about this together, do you perhaps see any other workaround then the one the mobile team is using now? thanks

Reply:
I think android-l10n automation was designed with the idea of supporting multiple branches, and that would solve this issue.

With that said, as long as the mobile team keeps track of removal consistently, the end result is the same.